src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
not sure I understand your suggestion - do you mean _struct dt_strided_ should contain struct dt_iov ? plus not sure why it "helps" strided to hold a cache...